superset/reports/commands/execute.py

```diff
@@ -305,6 +306,27 @@ def _update_query_context(self) -> None:
                 "Please try loading the chart and saving it again."
             ) from ex
 
+    def _get_log_data(self) -> Dict[str, Any]:
+        chart_id = None
+        dashboard_id = None
+        owners = json.loads(self._report_schedule.owners) or None
```

Comment:
will `self._report_schedule.owners` be a string?

Reply:
it should be an array